pkg/cluster/cluster.go

```diff
@@ -147,7 +147,7 @@ func GetCluster(connection *sdk.Connection, key string) (cluster *cmv1.Cluster,
 
 	// Try to find a matching subscription:
 	subsSearch := fmt.Sprintf(
-		"(display_name = '%s' or cluster_id = '%s' or external_cluster_id = '%s') and "+
+		"(name = '%s' or cluster_id = '%s' or external_cluster_id = '%s') and "+
```

Comment:
@hunterkepley this is an AMS field. Shouldn't change.
fixed in https://github.com/openshift-online/ocm-cli/pull/413

Reply:
cc @vkareh